### PR TITLE
fix: Refactored and removed the entire PackNGo library from metal-cli

### PR DESCRIPTION
**Pull request by codinja1188**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ This repository is [Maintained](https://github.com/packethost/standards/blob/mas
 
 [Equinix Metal](https://metal.equinix.com/) provides an API-driven bare metal platform that combines the automation experience of the cloud with the benefits of physical, bare metal servers.
 
-The Equinix Metal CLI wraps the [Equinix Metal Go SDK](https://github.com/packethost/packngo) allowing interaction with Equinix Metal platform from a command-line interface.
+The Equinix Metal CLI wraps the [Equinix Metal Go SDK](https://github.com/equinix/equinix-sdk-go) allowing interaction with Equinix Metal platform from a command-line interface.
```
Comment (ctreatma) [inline README.md]:
```suggestion
The Equinix Metal CLI wraps the [Equinix Go SDK](https://github.com/equinix/equinix-sdk-go) allowing interaction with Equinix Metal platform from a command-line interface.
```

File: docs/metal_facilities_get.md [hunk 2]
```diff
@@ -20,7 +20,9 @@ metal facilities get [flags]
 ### Options
 
 ```
-  -h, --help   help for get
+  -h, --help                     help for get
```
Comment (ctreatma) [inline docs/metal_facilities_get.md]: It looks like this leaked in from an older version of the facilities subcommand PR

File: internal/cli/root.go [hunk 3]
```diff
@@ -41,12 +40,11 @@ import (
 const (
 	envPrefix                  = "METAL"
 	configFileWithoutExtension = "metal"
-	debugVar                   = "PACKNGO_DEBUG"
+	debugVar                   = "METAL_DEBUG"
```
Comment (ctreatma) [inline internal/cli/root.go]: This should not change

Comment (codinja1188) [reply internal/cli/root.go]: sure, updated

File: internal/cli/root.go [hunk 4]
```diff
@@ -86,23 +64,10 @@ func NewClient(consumerToken, apiURL, Version string) *Client {
 	}
 }
 
-// This function provides backwards compatibility for the packngo
```
Comment (ctreatma) [inline internal/cli/root.go]: This comment should stay since we haven't settled on or introduced a new debug variable

**Review by ctreatma — APPROVED** [review]

**Pull request merged**